assemble/src/main/scripts/accumulo-client-build.jsh

```diff
@@ -0,0 +1,53 @@
+/*
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+*/
+  String accumuloProp = 
+      System.getProperty("user.dir").replace("/bin", "/conf/accumulo-client.properties");
+  String name; String zk; String principal; String token; 
+  AccumuloClient client = null; Authorizations auth = null;
+  
+  try {
+    // Does the accumulo properties file exists?
+    if (Files.exists(Paths.get(accumuloProp))) {
+      // Build Accumulo Client
+      client = Accumulo.newClient().from(accumuloProp).build();
+      
+      // Test Accumulo Client
+      auth = client.securityOperations().getUserAuthorizations(client.whoami());
+      if (auth != null) {
+        System.out.println("Accumulo Client was Successfully Built!");
+      System.out.println("Use " + "\"client\"" + " to interact with Accumulo \n");
+      }
+    }
+    else
+      System.out.println("accumulo-client.properties not found in conf directory \n");
+        
+  } catch (IllegalArgumentException illegalExcept) {
+    System.out.println("Build Error: "
+        + "Could not build Accumulo client with accumulo-client.properties \n"); 
+      
+  } catch (AccumuloException accumuloExcept) {
+    System.out.println("Connection Error: "
+        + "Ensure instance.name and instance.zookeeper are properly configured \n");
+        
+  } catch (AccumuloSecurityException secExcept) {
+    System.out.println( "Security Error: " 
+        + "Ensure auth.principal and auth.token are properly configured \n");
+        
+  } catch (NullPointerException nullExcept) {
+    System.out.println("Null Error: "
+        + "Could not build Accumulo client with accumulo-client.properties \n");  
+  }
```

Comment:
I think it would be nice to have a general catch all for unexpected runtime exceptions.

```suggestion
  } catch (RuntimeException e) {
    System.out.println("Error: "
        + "Could not build Accumulo client with "+accumloProp+" : "+e.getMessage()+" \n");  
  }
```

Reply:
Good point. That will definitely help the user know what specific issue occurred.

Reply:
Do we need to catch any exceptions at all? If we don't, what happens? I imagine that there will just be a stack trace... which is probably fine, and adequately informative. If we do need to print something, I think this is one of those few occassions where you might want to just do  `e.printStackTrace()`.

Reply:
@ctubbsii My initial thought was to output a simple error message so the user knows what error occurred without getting bogged down by a massive stack trace. However, after thinking about this more, it makes more sense to exclude the try and catch statements to provide users more information about what went wrong. Also with the classpath method implementation and omitting the testing portion, the exceptions above will not be thrown on startup. Therefore I believe we can omit the exception section all together.

Reply:
It may be nice to let the user know what config file was used, it makes things less magical.  Conveying this information does not have to be related to exceptions though.  Its nice if the user can get this info somehow, especially for cases like having multiple config files on the classpath and the user is not aware of them all.  Not sure of the best way to get the info to the user.  Always printing it is one possibility. Another option is printing it when an exception happens.  Are there any other options?

Reply:
Could do something like the following

```java

try {
  String accumuloProp = ...
  // attempt to create accumulo client
} catch (Exception e){
   System.out.println("Failed to create Accumulo client using properties file : "+accumuloProp);
   throw e;
}
```

Reply:
I think we could just output the file being used each time the user resets or starts up JShell. That way the user knows which properties file is being used at all times.